content/millennium/r4/base/individuals/related-person.md

```diff
@@ -181,7 +181,7 @@ _Implementation Notes_
 
 <%= definition_table(:related_person, :create, :r4) %>
 
-### Example
+### Example- Patient-Level RelatedPerson
```

Comment:
I think we will need a space before  '-' as well.

Reply:
updated: https://github.com/cerner/fhir.cerner.com/pull/689/commits/e42e12f2bcb15b514eeaf5cb5db3b6028d06a120#diff-2e663b902e2663663b5f937e76fc0abb32406c03fbea8ee6d4dc4a7f6eb56584R211